src/html/dom.pas

```diff
@@ -151,6 +151,8 @@       TElementChildrenNodeList = class (TNodeList)
         function GetItem(const Index: Integer): TNode; override;
         function GetFirstChild(): TNode; inline;
         function GetLastChild(): TNode; inline;
+        function GetFirstElementChild(): TElement inline;
```

Comment:
Hmm. I'd prefer to follow the pattern I used for NextElementSibling/PreviousElementSibling:

- Put it on Node
- Expose as a method, to indicate that it's expensive, instead of a property

Thoughts?

Reply:
> * Put it on Node
> * Expose as a method, to indicate that it's expensive, instead of a property

Yup, sounds right — made it so